src/richie/apps/core/templates/richie/base.html

```diff
@@ -63,9 +63,7 @@
                     <ul class="topbar__menu__list">
                         {% show_menu 0 100 0 0 "menu/header_menu.html" %}
                     </ul>
-                    <ul class="topbar__menu__list topbar__menu__list--aside">
-                        {% language_chooser "menu/language_menu.html" %}
-                    </ul>
+                    {% language_chooser "menu/language_menu.html" %}
```

Comment:
This is great, nice catch

The template boundaries were incorrect before IMO 👍